spec/compiler/macro/macro_methods_spec.cr

```diff
@@ -1960,6 +1960,35 @@ module Crystal
         assert_macro "x", %({{x.resolve?}}), [Crystal::Union.new(["Int32".path, "String".path] of ASTNode)] of ASTNode, "(Int32 | String)"
         assert_macro "x", %({{x.resolve?}}), [Crystal::Union.new(["Int32".path, "Unknown".path] of ASTNode)] of ASTNode, "nil"
       end
+
+      describe "#nilable?" do
+        it "without nil" do
```

Comment:
Please, use descriptive names, ditto elsewhere.

Reply:
How could it get more descriptive?  The method is called `nilable?` and it either can or cannot have nil.

Reply:
it's suppose to be read like a sentence - see other examples.